buckets/archive/tracker.go

```diff
@@ -169,11 +177,10 @@ func (t *Tracker) trackArchiveProgress(
 	jid string,
 	bucketRoot cid.Cid,
 	powInfo *mdb.PowInfo,
-) (bool, string, error) {
+) (time.Duration, string, error) {
```

Comment:
This now returns the duration that should be used for rescheduling. It will be different depending on if we have any deals in sealing state or not. `0` is interpreted as "don't reschedule".